runscripts/CI/awiesm/ollie/bootstrap.yaml

```diff
@@ -0,0 +1,53 @@
+general:
+    setup_name: "awiesm"
+    compute_time: "00:20:00"
+    initial_date: "1850-01-01T00:00:00"       # Initial exp. date
+    final_date: "1850-12-31T00:00:00"         # Final date of the experiment
+    project_base: !ENV ${PROJECT_BASE}
+    base_dir: "${general.project_base}/experiments"
+    nmonth: 0
+    nyear: 1
+    with_wiso: False
+    use_venv: False
+    clean_old_rundirs_except: 2
+    clean_old_rundirs_keep_every: 25
+    version: "2.1"
+    # Why is this twice??
+    model_dir: "${general.project_base}/model_codes/awiesm-2.1"
+    description: >
+      Basic AWIESM Test for PI
+
+awiesm:
+    postprocessing: false
+    # Why is this twice??
+    model_dir: "${general.project_base}/model_codes/awiesm-2.1"
+
+echam:
+    restart_rate: 1
+    restart_unit: "years"
+    input_sources:
+            jansurf: "/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_jan_surf.nc"
+            vgratclim: "/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VGRATCLIM.nc"
+            vltclim: "/work/ollie/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VLTCLIM.nc"
+    # Code for bootstrapping:
+    add_input_sources:
+        jansurf: "${project_base}/nonstandard_forcings/bootstrap_files/initial_unit24.nc"
+        target_orography: "${project_base}/nonstandard_forcings/bootstrap_files/target_unit24.nc"
+    add_input_files:
+            target_orography: target_orography
+    add_input_in_work:
+            target_orography: target_orography_echam6.nc
+    namelist_changes:
+            namelist.echam:
+                    submodelctl:
+                        lupdate_orog: True
```

Comment:
Are some of this things defaults that we want to include in ESM-Tools under a `bootstrap` switch or better not to.

Reply:
lines 36 until 43 could be done. Maybe even the add input sources: we could also do shortcuts for those files, but I will need to write extensive documentation, as generating them is not so straightforward